### PR TITLE
Make administration command integration tests pass for SQLite

### PR DESCRIPTION
**Pull request by noisersup**
# Description

- `commands_administration_compat_test.go`
- `commands_administration_test.go`
- `commands_authentication_test.go`
- `commands_diagnostic_test.go`

Closes #3131.

## Readiness checklist

- [ ] I added/updated unit tests (and they pass).
- [x] I added/updated integration/compatibility tests (and they pass).
- [ ] I added/updated comments and checked rendering.
- [x] I made spot refactorings.
- [ ] I updated user documentation.
- [ ] I ran `task all`, and it passed.
- [x] I ensured that PR title is good enough for the changelog.
- [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Labels, Project and project's Sprint fields.
- [x] I marked all done items in this checklist.

### CODE REVIEW

File: integration/commands_administration_test.go [hunk 1]
```diff
@@ -784,10 +797,10 @@ func TestCommandsAdministrationDataSize(t *testing.T) {
 	})
 }
 
-func TestCommandsAdministrationDataSizeErrors(t *testing.T) {
-	t.Parallel()
+func TestCommandsAdministrationDataSizeErrors(tt *testing.T) {
```
Comment (chilagrow) [inline integration/commands_administration_test.go:800]: This test fails on CI, could you check.

File: integration/commands_administration_compat_test.go [hunk 2]
```diff
@@ -118,10 +119,12 @@ func TestCommandsAdministrationCompatDBStatsWithScale(t *testing.T) {
 	} {
 		name, tc := name, tc
 
-		t.Run(name, func(t *testing.T) {
-			t.Helper()
+		t.Run(name, func(tt *testing.T) {
```
Comment (chilagrow) [inline integration/commands_administration_compat_test.go:122]: Also this test is not passing on CI.

File: integration/commands_administration_test.go [hunk 3]
```diff
@@ -138,8 +138,10 @@ func TestCommandsAdministrationCreateDropListDatabases(t *testing.T) {
 	assert.Equal(t, bson.D{{"ok", 1.0}}, res)
 }
 
-func TestCommandsAdministrationListDatabases(t *testing.T) {
-	t.Parallel()
+func TestCommandsAdministrationListDatabases(tt *testing.T) {
```
Comment (chilagrow) [inline integration/commands_administration_test.go:141]: Could you check why below tests are failing on CI for sqlite please :pray: 
`TestCommandsAdministrationCreateDropListDatabases`
`TestCommandsAdministrationServerStatusMetrics`
`TestCommandsAdministrationCompatCollStatsWithScale`
`TestCommandsAdministrationKillCursors`
`TestCommandsAdministrationListCollections`
`TestCommandsAdministrationBuildInfo`
`TestCommandsAdministrationCompatDBStatsWithScale`
`TestCommandsDiagnosticHostInfo`
`TestCommandsAdministrationDataSizeErrors`
`TestCommandsAuthenticationLogout`
`TestCommandsDiagnosticConnectionStatus`
`TestCommandsAdministrationCollStatsEmpty`
```


They fail like this
```
2023-08-02T15:45:16.9055686Z --- FAIL: TestCommandsAdministrationListCollections (0.15s)
2023-08-02T15:45:16.9055834Z     commands_administration_test.go:172: 
2023-08-02T15:45:16.9056147Z         	Error Trace:	/home/runner/work/FerretDB/FerretDB/integration/setup/listener.go:152
2023-08-02T15:45:16.9056721Z         	            				/home/runner/work/FerretDB/FerretDB/integration/setup/setup_compat.go:95
2023-08-02T15:45:16.9057300Z         	            				/home/runner/work/FerretDB/FerretDB/integration/setup/setup_compat.go:122
2023-08-02T15:45:16.9057935Z         	            				/home/runner/work/FerretDB/FerretDB/integration/commands_administration_test.go:172
2023-08-02T15:45:16.9058109Z         	Error:      	Received unexpected error:
2023-08-02T15:45:16.9059032Z         	            	[registry.go:76 metadata.NewRegistry] [registry.go:92 metadata.(*Registry).loadCollections] SQL logic error: no such table: _ferretdb_collections (1)
2023-08-02T15:45:16.9059271Z         	Test:       	TestCommandsAdministrationListCollections
```

**Review by chilagrow — COMMENTED** [review]
Could you check sqlite tests pass on those files please :pray:

**Review by chilagrow — APPROVED** [review]
Turns out the error looks like there is some flaky things with running sqlite test. All errors are the same so lets not block us

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Pull request merged**